test/test_listen_socket.cpp

```diff
@@ -203,3 +203,73 @@ TORRENT_TEST(partition_listen_sockets_op_ports)
 	TEST_EQUAL(eps.size(), 2);
 }
 
+bool listen_endpoint_cmp(aux::listen_endpoint_t const& l, aux::listen_endpoint_t const& r)
```

Comment:
Oh damnit. I added this before I decided to implement operator== and just forgot to delete it.